<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLient</class>
 <widget class="QWidget" name="CLient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLient</string>
  </property>
  <widget class="QTextEdit" name="quiz">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>681</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="connectAnswer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Answer</string>
   </property>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="checkBoxes">
     <item>
      <widget class="QRadioButton" name="a">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="b">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="c">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="d">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="connectMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu</string>
   </property>
   <widget class="QGroupBox" name="connectCreate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Create</string>
    </property>
    <widget class="QPushButton" name="create">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Quiz</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="code">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="numOfQuest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of questions</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="connectQuestion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>671</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Add question</string>
     </property>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="question">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>631</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="correct">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Correct answer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Question</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>671</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Connect to server</string>
    </property>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="server">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>localhost</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="port">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="join_game">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>671</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Join to the game</string>
    </property>
    <widget class="QPushButton" name="joinButton">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Join</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="join_code">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>31</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>660</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
